meta-resin-morty/recipes-containers/docker-disk/docker-resin-supervisor-disk.bb

```diff
@@ -0,0 +1,136 @@
+require ${RESIN_COREBASE}/recipes-containers/docker-disk/docker-disk.inc
```

Comment:
Can't we check if the interpreter is python3 or 2 and add decode() based on this? Maybe using  `if (sys.version_info > (3, 0))`